README.md

```diff
@@ -1,2 +1,55 @@
 # whole-body-estimators
 YARP-based estimators for humanoid robots.
+
+# Overview
+ - [:orange_book: Implementation](#orange_book-implementation)
+ - [:page_facing_up: Dependencies](#page_facing_up-dependencies)
+ - [:hammer: Build the suite](#hammer-build-the-suite)
+
+# :orange_book: Implementation
+The current implementations available in the `devices` folder include,
+- `baseEstimatorV1` includes a simple humanoid floating base estimation algorithm fusing legged odometry and IMU attitude estimation, aimed to be used along side walking controllers.
+
+![Floating Base Estimation Algorithm V1](doc/resources/fbeV1.png)
+
+
+# :page_facing_up: Dependencies
+* [YARP](http://www.yarp.it/): to handle the comunication with the robot;
+* [iDynTree](https://github.com/robotology/idyntree/tree/devel): to setup the floating base estimation algorithm;
+* [ICUB](https://github.com/robotology/icub-main): to use the utilities like low pass filters from the `ctrLib` library
+* [Gazebo](http://gazebosim.org/): for the simulation (tested Gazebo 8 and 9).
+
+  ## Optional Dependencies
+  * [walking-controllers](https://github.com/robotology/walking-controllers): to test the floating base estimation along side walking controllers
+
+It must be noted that all of these dependencies can be directly installed together in one place using the [robotology-superbuild](https://github.com/robotology/robotology-superbuild).
+
+
+# :hammer: Build the suite
+## Linux
+
+```sh
+git clone https://github.com/robotology/whole-body-estimators.git
+cd whole-body-estimators
+mkdir build && cd build
+cmake ../
+make
+[sudo] make install
+```
+Notice: `sudo` is not necessary if you specify the `CMAKE_INSTALL_PREFIX`. In this case it is necessary to add in the `.bashrc` or `.bash_profile` the following lines:
+``` sh
+export WBDEstimator_INSTALL_DIR=/path/where/you/installed
+export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${WBDEstimator_INSTALL_DIR}/lib/yarp
```

Comment:
In the future perhaps we can remove the need for this line by adding rpath handling.

Reply:
I think RPATH handling is already done in the CMakeLists. It's just that I didn't fully understand the implications of RPATH.

Reply:
I would like to understand,
does adding 
``` cmake
add_install_rpath_support(BIN_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}"
                          LIB_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
                          INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}"
                          DEPENDS ENABLE_RPATH
                          USE_LINK_PATH)
```
in your CMakeLists.txt add the generated dynamic libraries to the `LD_LIBRARY_PATH` automatically?

Reply:
Alright the documentation in `add_install_rpath_support` details this.

Reply:
Addressed in 82d978f1892d49beb5bdc29df0f343a153b2678a